docs/what-is-zarf.md

```diff
@@ -0,0 +1,225 @@
+
+Find out more about what downstream cluster service looks like in the "[Utility Mode](#utility-mode)" section.
+
+&nbsp;
+
+
+## How it's used (from 1,000 ft)
+
+### Building a package
+
+<a href="../.images/what-is-zarf/build-the-package.png">
+  <img alt="how it works" src="../.images/what-is-zarf/build-the-package.png" height="290" />
+</a>
+
+#### (0) - Identify software to-be-moved
+
+Zarf doesn't build software&mdash;it helps you distribute software that already exists.
+
+Zarf can pull from lots of places like [Docker Hub](https://hub.docker.com/), [Iron Bank](./ironbank.md), [GitHub](https://github.com/), local filesystems, etc. but you have to make sure that Zarf has a clear path & appropriate access credentials. Be sure you know what you want pack & how to access it before you Zarf.
+
+Find out more about the types of software Zarf can move in the "[What can be packaged?](#what-can-be-packaged)" section.
+
+#### (1) - Preparation
+
+In order to build a Zarf package, you need to prepare:
+
+- a "packaging" workstation &mdash; which must have the `zarf` [CLI tool installed](./workstation.md#just-gimmie-zarf), and
+
+- a `zarf.yaml` file &mdash; which tells the Zarf what you need packaged.
+
+Find some detailed uses of the `zarf.yaml` file in [our examples](../examples/).
+
+#### (2) - Package
+
+Actually making a Zarf package out of a `zarf.yaml` file is a matter of calling a single, simple command: `zarf package`.  You'll see a `zarf-package-*.tar.zst` file pop into existence afterward&mdash;that's your package.
+
+Find out more about that by calling the CLI for help, or check out an example package build in [our game example](../examples/game#package-the-game).
+
+&nbsp;
+
+
+### Shipping a package
+
+<a href="../.images/what-is-zarf/ship-the-package.png">
+  <img alt="how it works" src="../.images/what-is-zarf/ship-the-package.png" height="255" />
+</a>
+
+What this activity looks like is _very_ situationally specific so Zarf tries not to have an opinion&mdash;transfer Zarf packages between production & operating locations using whatever mechanisms are available.
+
+Have to burn your package to disk & "sneakernet" it? That works.
+
+Got an intermittent, super-secret satellite uplink you can use? Awesome.
+
+Can you make a direct network connection? Even better.
+
+Consider the art of the possible and use what you can&mdash;Zarf will continue to work, regardless.
+
+&nbsp;
+
+
+### Appliance Mode
+
+<a href="../.images/what-is-zarf/appliance-mode.png">
+  <img alt="how it works" src="../.images/what-is-zarf/appliance-mode.png" height="295" />
+</a>
+
+#### (0) - Package
+
+Move a Zarf release + your desired packages to your Zarf cluster machine.
+
+#### (1) - Zarf cluster
+
+Make your machine into a **single node** Zarf cluster with the command: `zarf init`.
+
+Recommended Zarf components: `management`.
+
+Deploy your package into the Zarf cluster with the command: `zarf deploy`.
```

Comment:
same here it's actually `zarf *zarffilename*` or `zarf package deploy` to get the menu

Reply:
Definitely forgot the "package" part in there too--good catch.  Should read `zarf package deploy` for sure.